go.mod

```diff
@@ -166,28 +166,28 @@ require (
 	go.etcd.io/etcd/etcdutl/v3 v3.5.15
 	go.uber.org/zap v1.27.0
 	go4.org/netipx v0.0.0-20231129151722-fdeea329fbba
-	golang.org/x/net v0.26.0
-	golang.org/x/oauth2 v0.21.0
-	golang.org/x/sync v0.7.0
-	golang.org/x/sys v0.22.0
-	golang.org/x/term v0.21.0
-	golang.org/x/text v0.16.0
-	golang.org/x/time v0.5.0
+	golang.org/x/net v0.28.0
+	golang.org/x/oauth2 v0.22.0
+	golang.org/x/sync v0.8.0
+	golang.org/x/sys v0.23.0
```

Comment:
this won't work https://github.com/golang/go/issues/68761

Reply:
First time I had to use `go mod graph | grep 'x/sys@v0.23.0`. The chain goes like:

```
golang.org/x/crypto -> golang.org/x/sys
golang.org/x/tools  -> golang.org/x/net -> golang.org/x/sys
golang.org/x/term   -> golang.org/x/sys
```